task_1.py

```diff
@@ -0,0 +1,118 @@
+print('Добро пожаловать в Телефонный справочник!')
+filename = "contacts.txt"
+myfile = open(filename, "a+")
+myfile.close
```

Comment:
Если вы хотите таким образом создать файл, то можно сделать это проще, без создания переменной
```suggestion
open(filename, "a").myfile.close()
```